### ISSUE TITLE
Menu should be enlargable

### ISSUE THREAD

**Issue by flexiondotorg** [ubuntu-mate/mate-menu#18]
Originally reported by: **Luca Moscato (Bitbucket: [tinom](https://bitbucket.org/tinom), GitHub: [tinom](https://github.com/tinom))**

----------------------------------------

Environment:
Ubuntu 16.04 LTS - Virtualized with Virtualbox
Mate installed after fresh installation and updated to Mate 1.16 via Ubuntu Mate repo
Mate menu installed and placed in top bar

I'm using Mate menu in italian, don't know if in other languages the issue is reproducible, supposed yes.

Menu should have the possibility to be resizable (or at least a bit bigger) to avoid text truncation in second and third column

----------------------------------------
- Bitbucket: https://bitbucket.org/ubuntu-mate/mate-menu/issue/18

**Comment by buzzra** [comment]
I second this request. I also do not consider it a "minor" improvement. I can't believe it has been on the issue list for over a year and a half!

**Comment by tecan** [comment]
basically wider with more columns ?

**Comment by buzzra** [comment]
> basically wider with more columns ?

No, actually resizable to as large or small as desired. I would like to be able to enlarge it so that the full title of the app icons is visible for all apps.
